database/migrations/2019_03_35_000001_create_logger_tables.php

```diff
@@ -58,7 +58,13 @@ public function up(): void
                 ->onDelete('RESTRICT');
         });
 
-        DB::statement('ALTER TABLE `audit_activities` ADD `ip_address` VARBINARY(16) AFTER `type`');
+        if (DB::getDriverName() === 'pgsql') {
+            // PostgreSQL-specific SQL
+            DB::statement('ALTER TABLE audit_activities ADD ip_address INET');
+        } else {
+            // MySQL-specific SQL
+            DB::statement('ALTER TABLE `audit_activities` ADD `ip_address` VARBINARY(16) AFTER `type`');
+        }
```

Comment:
Just wondering, since this change adds a change specific to Postgres and would mean we're switching to a new DB, is it okay that this is editing an existing migration file? It wouldn't break existing projects that use this?

Reply:
This isn't switching to a new DB, just adding support for it.
There's 3 possibilities:

Existing projects - Will have already ran this migration, we don't need to worry since they're all non-PostGres.
New Projects that use PostGres - Will run the first if statement.
New projects that don't use PostGres - Will run the else statement.

Reply:
I was actually thinking of a different possibility where we'd have an existing project that isn't using Postgres but is making a switch to Postgres. But I think that case is probably handled by the second case you mentioned

I just usually don't see PRs where we change an existing migration rather than adding a new one, which is why I was a bit cautious. But I think it makes sense for what's being done here

Reply:
Oh I didn't think about the case of a project that is switching to PostGres. 
When we switch to PostGres do we rerun all migrations? I'm not sure the process of switching databases.

Reply:
I think we do, at least for alpha and staging. I'm not sure what that's like in production. When we switched over to Postgres in UL, we started from a fresh database in both alpha and staging.

Reply:
Yeah I think if a project is in Production, we probably wouldn't bother trying to switch its database over.

Reply:
I think this modification is probably fine as long as we tag the release appropriately. This change isn't going to impact existing users of this package, however it does open up the question of whether this package should be publishing the migrations rather than just loading the migrations like it is. 

@iBotPeaches Do you have any recollection as to why we aren't publishing the migrations?

Reply:
I don't think publishing migrations was the default when this was written. Like it was built around time of Passport, etc that automatically ran migrations on pkg install.